samples/tls/monitoring/README.md

```diff
@@ -33,7 +35,7 @@ my_repository_path/monitoring/bin/deploy_monitoring_cluster.sh
 any sample hostnames with hostnames for your deployment. Specifically, you must replace
 `host.cluster.example.com` with the name of the ingress node. Often, the ingress node is the cluster master node, but your environment might be different.
 
-* If you plan on using ingress, you must manually populate these Kubernetes secrets with TLS certificates before you deploy cluster monitoring:
+* If you are using ingress, you must manually populate these Kubernetes secrets with TLS certificates before you deploy cluster monitoring:
```

Comment:
It is possible to use ingress with or without setting the TLS values (certs) in the ingress object. So ingress can be HTTP or HTTPS.